scvi/model/_condscvi.py

```diff
@@ -106,6 +104,8 @@ def get_vamp_prior(
             AnnData object used to initialize the model.
         p
```

Comment:
looks like p has been removed here. Do we want it to be an option then, or do we want just the clustering?

I think clustering should be more robust, so I'd lean towards removing the option p if that makes sense to you. You should have only resolution (or p is you use k-means; but not both)

Reply:
p is still in use, sampling p cells from the constructed clusters based on cluster_probability (it's also beneficial for performance). Leiden tends to find small outlier clusters that are weighted higher when not sampling.

Reply:
Implemented, it's 15 clusters and there is some minor speed gain